etsin_finder/frontend/js/components/qvain/datasets/formatAge.js

```diff
@@ -35,7 +35,11 @@ const formatAge = (currentTime, eventTime) => {
     // More than a month ago, compare by months
     const monthsSinceCreation = timestampCurrentTime.diff(timestampEvent, 'months')
 
-    if (monthsSinceCreation >= 1) {
+    if (secondsSinceCreation >= 2160000 && monthsSinceCreation === 0) { // 1 month
+      formattedDate = `${timestampCurrentTime.diff(timestampEvent, 'days')} ${translate(
+      'qvain.datasets.tableRows.dateFormat.days'
+    )}`
+    } else if (monthsSinceCreation >= 1) {
```

Comment:
Just noticed that the function never returns more than "1 month ago" because if `monthsSinceCreation >= 1`, the later cases (`monthsSinceCreation >= 10` etc.) never happen. 

I'd move `const monthsSinceCreation = timestampCurrentTime.diff(timestampEvent, 'months')` to the beginning of the function and then replace `secondsSinceCreation < 2160000` with `monthSinceCreation < 1`. Then rewrite the remaining cases after that so that the end result makes sense and there are no redundant cases.

Reply:
This is now rewritten, thanks for comment